src/shared/lib/classNames.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for classnames
```

Comment:
Interesting, could be useful to mention this in some sort of style guide or the wiki